sql/core/src/main/scala/org/apache/spark/sql/sources/commands.scala

```diff
@@ -290,6 +298,9 @@ private[sql] abstract class BaseWriterContainer(
     setupIDs(0, 0, 0)
     setupConf()
 
+    ContextUtil.getConfiguration(job).set(
+      "spark.sql.sources.writeJobUUID", uniqueWriteJobId.toString)
```

Comment:
Why we use a parquet method at here? Are we expecting that `getConfiguration` does not exist in some versions of `Job`?